scopes/docs/docs/docs.preview-definition.ts

```diff
@@ -33,6 +33,6 @@ export class DocsPreviewDefinition implements PreviewDefinition {
    */
   async getModuleMap(components: Component[]): Promise<ComponentMap<AbstractVinyl[]>> {
     const map = this.docs.getDocsMap(components);
-    return map.filter((value) => value.length !== 0);
+    return map;
```

Comment:
no need to hide components with no previews.
also verified every usage handles `[]` correctly